apps/files/src/components/Collaborators/Collaborator.vue

```diff
@@ -4,7 +4,7 @@
     <div class="files-collaborators-collaborator-information uk-flex uk-flex-wrap uk-flex-middle">
       <oc-spinner v-if="loading" key="collaborator-avatar-spinner" uk-spinner="ratio:1.6" class="uk-margin-small-right" :aria-label="$gettext('Loading')"/>
       <div v-else key="collaborator-avatar-loaded">
-        <oc-avatar v-if="avatar" :src="avatar" class="uk-margin-small-right" width=50 height=50 />
+        <oc-avatar v-if="collaborator.info.share_type === '0'" :src="avatar" class="uk-margin-small-right" size=50 :userName="collaborator.displayName" />
```

Comment:
This should still be `width` instead of `size`, right?!

Reply:
indeed :-/